rosidl_generator_c/include/rosidl_generator_c/service_type_support.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2014 Open Source Robotics Foundation, Inc.
```

Comment:
Let's update the copyright year as long as we're making a new file.

I'll add an issue to the backlog for auditing our copy year usage across the project since we're actively updating it.